examples/examples_test.sh

```diff
@@ -110,7 +110,7 @@ declare -A EXPECTED_SERVER_OUTPUT=(
     ["features/authz"]="unary echoing message \"hello world\""
     ["features/cancellation"]="server: error receiving from stream: rpc error: code = Canceled desc = context canceled"
     ["features/compression"]="UnaryEcho called with message \"compress\""
-    ["features/customloadbalancer"]="serving on localhost:20000"
+    ["features/customloadbalancer"]="serving on localhost:50051"
```

Comment:
I would like to see an `EXPECTED_CLIENT_OUTPUT` entry for this example in this file or else we aren't actually testing it.  It also needs to be added to `EXAMPLES`, right?

Reply:
Oops I thought I was looking at the full diffs, but I was just looking at one of many recent commits.  Sorry.